test/cli/help/help.out

```diff
@@ -91,7 +91,15 @@ Usage:
       --enable-experimental-lsp-quick-fix
                                 Enable experimental LSP feature: Quick Fix
       --enable-all-experimental-lsp-features
-                                Enable every experimental LSP feature.
+                                Enable every experimental LSP feature
+                                (WARNING: Experimental features may be
+                                destabilizing/crashing; not recommended for non-developers.
+                                Users wishing to beta-test LSP features should
+                                consider `--enable-all-beta-lsp-features`,
+                                instead)
```

Comment:
This is really long. What do you think about:

```
Enable every experimental LSP feature. Warning: these might crash! Consider `--enable-all-beta-lsp-features` instead.
```

Reply:
I think that the goal here should be to discourage use of this flag from people who don't have a feedback channel to the Sorbet development team, which is why I wanted to make it a bit scary and specifically use the term "non-developer" here.

The scenario I wanted to avoid is an end-user confusing this flag with late-stage beta-testing, having a bad experience, and not being equipped to understand why (or worse, forgetting that they enabled the flag and languishing in that bad state).

Alternative: what if we made the flag hidden/unadvertised, so that only developers reading the code see it and know to enable it?